swap_meet/clothing.py

```diff
@@ -1,2 +1,11 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    def __init__(self, year=None, id=None, fabric='Unknown', condition=0):
+        super().__init__(year = year, id=id, condition=condition)
```

Comment:
[suggestion] Keep spacing consistent

```suggestion
        super().__init__(year=year, id=id, condition=condition)
```